Mathlib/MeasureTheory/Function/LpSeminorm/Basic.lean

```diff
@@ -465,14 +493,29 @@ theorem eLpNorm_mono_real {f : α → F} {g : α → ℝ} (h : ∀ x, ‖f x‖
     eLpNorm f p μ ≤ eLpNorm g p μ :=
   eLpNorm_mono_ae_real (Eventually.of_forall fun x => h x)
 
+theorem eLpNormEssSup_le_of_ae_enorm_bound {f : α → ε} {C : ℝ≥0} (hfC : ∀ᵐ x ∂μ, ‖f x‖ₑ ≤ C) :
+    eLpNormEssSup f μ ≤ C :=
+  essSup_le_of_ae_le (C : ℝ≥0∞) hfC
+
 theorem eLpNormEssSup_le_of_ae_nnnorm_bound {f : α → F} {C : ℝ≥0} (hfC : ∀ᵐ x ∂μ, ‖f x‖₊ ≤ C) :
     eLpNormEssSup f μ ≤ C :=
   essSup_le_of_ae_le (C : ℝ≥0∞) <| hfC.mono fun _x hx => ENNReal.coe_le_coe.mpr hx
 
+-- XXX: better naming suggestions welcome
```

Comment:
not sure what the point of this lemma is, compared to the lemma above, which is nicer. You could use `eLpNormEssSup_le_of_ae_bound_enorm` or something...

Reply:
I don't remember if I actually used this lemma for anything. I have removed it for now; if a dependent PR needs it, I can always bring it back.